### PR TITLE
Add documentation for ADRs

### PR DESCRIPTION
**Pull request by fkneeland-figure**
## Description

We currently don't have a place for ADRs to be put and have broken links in our documentation.

closes: #158 

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/provenance-io/explorer/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote unit and integration
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ Other notes:
 When proposing an architecture decision for the SDK, please create an [ADR](./docs/architecture/README.md)
 so further discussions can be made. We are following this process so all involved parties are in
 agreement before any party begins coding the proposed implementation. If you would like to see some examples
-of how these are written refer to [Tendermint ADRs](https://github.com/tendermint/tendermint/tree/main/docs/architecture)
+of how these are written refer to [Tendermint ADRs](https://github.com/tendermint/tendermint/tree/master/docs/architecture)
```
Comment (fkneeland-figure) [inline CONTRIBUTING.md:70]: This link was broken and this fixed it.

**Review by iramiller — APPROVED** [review]
Nice.

File: docs/architecture/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Architecture Decision Records (ADR)
+
+This is a location to record all high-level architecture decisions in the tendermint project.
```
Comment (crussell-figure) [inline docs/architecture/README.md:3]: This was probably a copy-pasta, but should be updated to be explorer specific.

Comment (fkneeland-figure) [reply docs/architecture/README.md:3]: 😞 Here is a pr to fix that issue: https://github.com/provenance-io/explorer-service/pull/171/files @crussell-figure

**Pull request merged**